setup.py

```diff
@@ -57,6 +57,10 @@
 AUTHOR = 'Bioinformatics Laboratory, FRI UL'
 AUTHOR_EMAIL = 'info@biolab.si'
 URL = 'http://orange.biolab.si/'
+PROJECT_URLS = {
+    'Documentation': 'https://orange3.readthedocs.io',
```

Comment:
Maybe the doc link should point to https://orangedatamining.com/docs
The one above might change in the future and covers only part of the documentation.